### PR TITLE
feat: remove devAssistant

### LINKED ISSUE #291: Remove DevAssistant entirely
So there is a page with DevAssisatnt at Developer Portal that says "DevAssistant is obsolete, use it for reference only"

I don't know what kind of reference is that supposed for, but maybe it's time to get rid fo that page entirely? Or at lest the big logo and link at https://developer.fedoraproject.org/tools.html

http://developer.fedoraproject.org/tools/devassistant/about.html

https://github.com/developer-portal/content/issues/194
https://github.com/developer-portal/content/issues/196

### PR DESCRIPTION
**Pull request by gombosg**
devAssistant is dead :(

Fixes #291

### CODE REVIEW

**Comment by pvalena** [comment]
Thanks!
Could you also remove the entries and references to it from `tech/languages/java/java-build-tools-installation.md`?

**Comment by gombosg** [comment]
I reworked and expanded them instead if you're OK with that.

File: tech/languages/java/java-build-tools-installation.md [hunk 1]
```diff
@@ -24,22 +24,46 @@ Last but not least, still quite popular duo [Ant+Ivy](http://ant.apache.org/ivy/
 $ sudo dnf install ant apache-ivy
 ```
 
-## DevAssistant is obsolete, use it for reference only
+## Creating a Maven project
 
-You can get started with Maven by using [DevAssistant](/tools/devassistant/about.html) to create a simple project, simply type (after you have DevAssistant setup and installed):   
+### Using `mvn` command-line tool
+
+Use the command from the [Maven quick start guide](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html):
```
Comment (pvalena) [inline tech/languages/java/java-build-tools-installation.md:31]: This sounds great! But don't we have the `mvn` packaged in Fedora?

If it's about the guide, please put the reference after the `quick start` "text" instead.

File: tech/languages/java/java-build-tools-installation.md [hunk 2]
```diff
@@ -24,22 +24,46 @@ Last but not least, still quite popular duo [Ant+Ivy](http://ant.apache.org/ivy/
...
+This directory is where your compiled artifacts, or generated outputs (from builds) are stored. Its best to think of this directory as a temp space, used to store the outputs from a build. 
+
+### Creating a git repo
 
-The `target` directory is created as part of your initial project build, triggered by DevAssistant command above. This directory is where your compiled artifacts, or generated outputs (from builds) are stored. Its best to think of this directory as a temp space, used to store the outputs from a build. 
+This project folder is not a Git repository yet. Create a git repo:
+```
+$ cd my-app
+$ git init
```
Comment (pvalena) [inline tech/languages/java/java-build-tools-installation.md:54]: I plan to do a standalone `git` quickstart, so I'll move most of this later on, leaving just a reference.

**Review by pvalena — CHANGES_REQUESTED** [review]
Looks good! My only question would be how to get the `mvn` executable.

Pointing simply to another quick start guide is also fine with me (instead of duplicating the content here).

Is there anything benefitial here that 's not in the referenced guide?

File: tech/languages/java/java-build-tools-installation.md (hunk 1 above)
Comment (pvalena) [reply tech/languages/java/java-build-tools-installation.md:31]: Let's address this in standalone PR if it needs to be changed (**is mvn in Fedora?**).

**Pull request merged**